src/index.html

```diff
@@ -294,26 +294,23 @@ <h2>week of ...</h2>
           </div>
         </section>
       </section>
-      <section class="single-recipe-view hidden">
-        <h1>Recipe Name</h1>
-        <div>
-          <button type="button" class="single-recipe-view__add-to-favorites-button">♡</button>
-          <p>ADD TO FAVORITES</p>
-        </div>
-        <div>
-          <button type="button" class="single-recipe-view__add-to-week-button"></button>
-          <p>ADD THIS WEEK'S RECIPES</p>
-        </div>
-        <p class="single-recipe-view__total-cost">💲COST: </p>
-        <img src= class="single-recipe-view__recipe-image" alt=>
-
-        <div class="single-recipe-view__ingredients">
-          <h2>INGREDIENTS</h2>
-          <div class="ingredients__ingredient-list"></div>
+      <section class="single-recipe-view hidden" id="singleRecipeView">
+        <div class="single-recipe-view__recipe-information">
+          <div class="recipe-information__container">
+            <h1 class="recipe-title" id="recipeName">Recipe Name</h1>
+            <button type="button" class="recipe-button add-to-favorites-button">♡ ADD TO FAVORITES</button>
+            <button type="button" class="recipe-button add-to-week-button">🗓 ADD TO THIS WEEK'S</button>
+            <h2>INGREDIENTS</h2>
+            <div class="ingredient-list" id="recipeIngredients"></div>
+            <p class="total-cost">COST:💲<span id="recipeCost"></span></p>
```

Comment:
You could use [this icon](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aattach_money%3A) for the dollar sign, too.